my_game.py

```diff
@@ -316,11 +316,8 @@ def setup(self):
         )
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
```

Comment:
Let's put the number of Asteroids in a global variable `ASTEROIDS_PR_LEVEL = 5`

Reply:
Got it! :)

Reply:
You forgot the the S in ASTEROIDS

Reply:
Nu er der et S